### PR TITLE
Update jackson to 2.16.2

### PR DESCRIPTION
**Pull request by wendigo**
Changelog: https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.16.2

### CODE REVIEW

**Comment by wendigo** [comment]
cc @electrum

**Comment by wendigo** [comment]
ping @electrum

File: CHANGES.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Airbase 152
   - kotlin 1.9.23 (from 1.9.22)
   - jakarta.xml.bind-api 4.0.2 (from 4.0.1)
   - ASM 9.6 (from 9.5)
+  - jackson 2.16.2 (from 2.16.1)
```
Comment (electrum) [inline CHANGES.md:10]: Move to 153

**Comment by wendigo** [comment]
Replaced by https://github.com/airlift/airbase/pull/402

**Pull request closed without merging**